community/README.md

```diff
@@ -39,7 +39,7 @@ within 24 hours of the meeting.
 
 ## Contributing to Argo
 
-Read and abide by the [Argo Code of Conduct](https://github.com/argoproj/argo/blob/master/CODE_OF_CONDUCT.md).
+Read and abide by the [CNCF Community Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
 
```

Comment:
Link to the proposed Argo Code of conduct as well?

Reply:
Thinking we should just use the CNCF code of conduct to avoid duplication/confusion.

Reply:
That will make it neat and clean.